### PR TITLE
issue #10 - color

### PR DESCRIPTION
**Pull request by estelled**

### CODE REVIEW

File: src/app/contribution-graph/contribution-graph.component.ts [hunk 1]
```diff
@@ -134,4 +137,60 @@ export class ContributionGraphComponent implements OnInit {
...
+    const ratio = activityCount / totalActivityCount * 100;
+    let value = 0;
+    if (ratio < 25) {
+      value = 1;
+    } else if (ratio < 50) {
+      value = 2;
+    } else if (ratio < 75) {
+      value = 3;
+    } else {
+      value = 4;
```
Comment (zermelozf) [inline src/app/contribution-graph/contribution-graph.component.ts]: You could define `let value = 4;` and then remove the last else condition.

File: src/app/contribution-graph/contribution-graph.component.ts [hunk 2]
```diff
@@ -134,4 +137,60 @@ export class ContributionGraphComponent implements OnInit {
...
+    }
+    return value;
+  }
+
+
+  circleColor(i: number, j: number) {
+    const colorTable = this.colorScheme;
```
Comment (zermelozf) [inline src/app/contribution-graph/contribution-graph.component.ts]: This feels a bit unnecessary maybe. You could just use `this.colorScheme` directly in the code.

File: src/app/contribution-graph/contribution-graph.component.ts [hunk 3]
```diff
@@ -134,4 +137,60 @@ export class ContributionGraphComponent implements OnInit {
...
+    const id =  this.idCalc(i, j);
+    const date = this.id2date(id);
```
Comment (zermelozf) [inline src/app/contribution-graph/contribution-graph.component.ts]: I think line `185` and line `186` are used in several places in the code. Maybe it'd be good to create a function `getDate(col: number, row: number)` that directly returns the date?

**Review by zermelozf — APPROVED** [review]
LGTM.

**Comment by estelled** [comment]
@zermelozf Just included your suggested updates. can you merge it ?

**Pull request merged**